test/NuGet.Core.Tests/NuGet.Packaging.Test/PackageBuilderTest.cs

```diff
@@ -160,8 +214,8 @@ public void CreatePackageWithNuspecIncludeExclude()
             var net45 = new PackageDependencyGroup(new NuGetFramework(".NETFramework", new Version(4, 5)), dependencies45);
             builder.DependencyGroups.Add(net45);
 
-            var net46 = new PackageDependencyGroup(new NuGetFramework(".NETFramework", new Version(4, 6)), dependencies46);
-            builder.DependencyGroups.Add(net46);
+            var net50win = new PackageDependencyGroup(new NuGetFramework(".NETCoreApp", new Version(5, 0), "windows", new Version(0, 0)), dependencies50);
```

Comment:
The most full proof test to ensure net50 can be packed is to add a test in https://github.com/NuGet/NuGet.Client/blob/dev/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs. 

Add the test only for net5.0 projects.